myproject2/myapp2/management/commands/create_client.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+
+from .models import Client
```

Comment:
Попробуйте импортировать так
```suggestion
from myapp2.models import Client
```